sqflite/lib/src/sentry_database.dart

```diff
@@ -66,6 +93,7 @@ class SentryDatabase extends SentryDatabaseExecutor implements Database {
       try {
         await _database.close();
 
+        dbName = null;
```

Comment:
m: I dont think you need to clean the name. We could use it to log if the user try to use a closed databased and an exception is thrown.

Reply:
sounds good 👍